src/components/chef-card.tsx

```diff
@@ -0,0 +1,20 @@
+import Link from "next/link";
+
+type Props = {
+  routingUrl: string;
+  imageUrl: string;
+  imageText: string;
+};
+
+// 注目シェフ用のカード
+export const ChefCard = (props: Props) => {
+  const { routingUrl, imageUrl, imageText } = props;
```

Comment:
```suggestion
export const ChefCard = ({ routingUrl, imageUrl, imageText }: Props) => {
```

Propsの分割代入のほうが可読性上がると思いました！